src/bindings/python/flux/accounting/job_archive_interface.py

```diff
@@ -394,16 +394,13 @@ def calc_usage_factor(
     acct_conn.commit()
 
     # write historical usage to first column in job_usage_factor_table
-    update_stmt = (
-        "UPDATE job_usage_factor_table SET usage_factor_period_0="
-        + str(usage_user_historical)
-        + " WHERE username='"
-        + user
-        + "' AND bank='"
-        + bank
-        + "'"
-    )
-    acct_conn.execute(update_stmt)
+    update_stmt = """
+        UPDATE job_usage_factor_table
+        SET usage_factor_period_0=?
+        WHERE username=?
+        AND bank=?
+        """
+    acct_conn.execute(update_stmt, (str(usage_user_historical), user, bank,))
```

Comment:
Does this require the `str` call? Or can it be dropped?

Reply:
This does not - thanks for pointing that out. I will append a `fixup` commit to address this.

Reply:
LGTM! Feel free to squash that fixup commit.